cmd/up.go

```diff
@@ -205,3 +221,46 @@ func clusterUpLocal(ctx *cli.Context) error {
 	_, _, _, _, _, err = ClusterUp(context.Background(), rkeConfig, nil, hosts.LocalHealthcheckFactory, nil, true, "", false, false)
 	return err
 }
+
+func clusterUpDind(ctx *cli.Context) error {
+	clusterFile, filePath, err := resolveClusterFile(ctx)
+	if err != nil {
+		return fmt.Errorf("Failed to resolve cluster file: %v", err)
+	}
+	clusterFilePath = filePath
+
+	rkeConfig, err := cluster.ParseConfig(clusterFile)
+	if err != nil {
+		return fmt.Errorf("Failed to parse cluster file: %v", err)
+	}
+
+	rkeConfig, err = setOptionsFromCLI(ctx, rkeConfig)
+	if err != nil {
+		return err
+	}
+
+	disablePortCheck := ctx.Bool("disable-port-check")
```

Comment:
do you think it would make sense to extract config setup + dind setup to it's own methods? So clusterupdind would look something like:

* read/setupConfig
* deploy dind
* cluster up

Reply:
sure will fix that